tests/end2end/features/qutescheme.feature

```diff
@@ -171,6 +171,15 @@ Feature: Special qute:// pages
         And I wait until qute://pyeval/ is loaded
         Then the page should contain the plaintext "ZeroDivisionError"
 
+    Scenario: Running :pyveal with --file using a file that exists as python code
+        When I run :debug-pyeval --file (testdata)/misc/pyeval_file.py
+        Then the message "Hello World" should be shown
+        And "pyeval output: No error" should be logged
+
+    Scenario: Running :pyeval --file using a non existing file
+        When I run :debug-pyeval --file nonexestentfile
+        Then the error "[Errno 2] No such file or directory: 'nonexistentfile'" should be shown
```

Comment:
I'm not sure this will work on Windows, the error message might be slightly different. But we'll [soon see](https://ci.appveyor.com/project/qutebrowser/qutebrowser/build/master-2633/job/5w457lyo41ww4aew) thanks to AppVeyor :wink: